server/plugins/PanUGens.cpp

```diff
@@ -1304,12 +1304,12 @@ void PanAz_Dtor(PanAz* unit) {
 void PanAz_next_ak(PanAz* unit, int inNumSamples) {
     float pos = ZIN0(1);
     float level = ZIN0(2);
-    float width = ZIN0(3);
+    double width = ZIN0(3);
     float orientation = ZIN0(4);
 
     int numOutputs = unit->mNumOutputs;
     float rwidth = sc_reciprocal(width);
-    float range = numOutputs * rwidth;
+    double range = numOutputs * rwidth;
     float rrange = sc_reciprocal(range);
```

Comment:
the precision loss is in the float version of sc_reciprocal. It was not due to range or rrange being a float, it was just because sc_reciprocal has two different algorithms for float and doubles. I thought we didn't need to have rrange as double as well, but maybe I'm wrong. Anyway, I can change it to double for consistency.

Reply:
yes, but there is now an additional loss of precision by assigning `double` to `float` which needs to be addressed - either by casting (to express clearly that it is what you intend) or by increasing the precision. I would recommend using higher precision in this case

Reply:
Actually, I would go for static_casts, since they make more explicit that we need double precision only temporary: i.e. only as argument to sc_reciprocal, to select the right algorithm.
Other than that, floats are enough IMO, because we are using those values to index into a table that has 4096 items anyway

Reply:
But please let me know if you would prefer doubles everywhere (relevant) instead

Reply:
Ops, I haven't seen your previous comment before I wrote mine... again, if you think doubles are better than this static_casts I did, let me know and I will change them

Reply:
if it still works, the new code clearly expresses your intent and stops the compiler warnings, which were my main concerns.

I suggested using `double` because in general it's better to have higher precision for as long as possible to reduce rounding errors, so as long as it still works?

Reply:
It still works the same. Again, I think floats are enough in this case
since in the end these values are used to index in a table with 4096
elements. So, I thought it would be good to make my change as "local" as
possible.